core/src/helpers.ts

```diff
@@ -28,6 +28,33 @@ const upperFirst = (value: string): string =>
 const lowerFirst = (value: string): string =>
   value.charAt(0).toLowerCase() + value.slice(1);
 
+// eslint-disable-next-line  @typescript-eslint/no-explicit-any
```

Comment:
yeah, I know - any help/suggestions for typing this in a more specific manner would be appreciated.